bin/lib/deploy_utils

```diff
@@ -75,9 +75,19 @@ generate_manifest() {
     manifest=$(echo "${manifest}" | bosh-cli int - --vars-file="${bosh_environment}/${deployment_name}-vars.yml")
   fi
 
-  manifest=$(echo "${manifest}" | bosh-cli int - \
-      --vars-file="${bosh_environment}/director.yml" \
-      --var=deployment_name="${deployment_name}")
+  if bosh-cli int "${bosh_environment}/director.yml" --path='/http_proxy'; then
```

Comment:
It gets kinda weird because we source this utility file from our other scripts. Maybe @alex-slynko or @jhvhs has ideas.